README.md

```diff
@@ -55,28 +55,6 @@ Logs in the local Docker client to one or more Amazon ECR Private registries or
           docker push $REGISTRY/$REPOSITORY:$IMAGE_TAG
 ```
 
-#### Login to Amazon ECR Private, then build and push a Docker image masking the password:
```

Comment:
I want to push a follow-up PR to make masking the default but it's a breaking change.  I think it begs the broader question do we even want this escape hatch as it allows users to essentially leak CI creds via PR's.

Reply:
Yep, agreed. Is there a way to do that while still allowing users to run jobs based off an image stored in ECR Private? I'm not entirely convinced we need to prevent this entirely, as my understanding is that the creds would only be visible if debug logging was enabled. Doing this in a private GH repo, for example, might be a tradeoff some people are willing to make, as it would only expose the credentials to members of their organization and the creds themselves expire after 12 hours.

Reply:
Unfortunately, once you mask the password, it cannot be shared. I think the overhead of running this action (and a AWS login action) is so light that it's not worth the risk of leaking CI user passwords (often privileged).

The attack vector I am thinking of is someone can use the leaked credentials to overwrite an image (if tag immutability is disabled) and ship a malicious binary to prod.

I personally would like to make a breaking change that disables this behaviour entirely and people can use the old action if they want this insecure behaviour.